test/unit/DetectChanges/BCBreak/ClassBased/TraitUsePropertyRemovedTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RoaveTest\BackwardCompatibility\DetectChanges\BCBreak\ClassBased;
+
+use PHPUnit\Framework\TestCase;
+use Roave\BackwardCompatibility\Change;
+use Roave\BackwardCompatibility\DetectChanges\BCBreak\ClassBased\PropertyRemoved;
+use Roave\BetterReflection\BetterReflection;
+use Roave\BetterReflection\Reflection\ReflectionClass;
+use Roave\BetterReflection\Reflector\ClassReflector;
+use Roave\BetterReflection\SourceLocator\Type\StringSourceLocator;
+
+use function array_map;
+use function iterator_to_array;
+
+final class TraitUsePropertyRemovedTest extends TestCase
```

Comment:
I think this should be part of the `PropertyRemovedTest` (discussed in https://github.com/Roave/BackwardCompatibilityCheck/pull/269/files#r519199158)

Reply:
I've removed this file and moved the tests to the propertyRemoved test file